Assets/Scripts/RelayManager.cs

```diff
@@ -25,31 +25,7 @@ public enum RelayType
     [SerializeField]
     private int maxNumberOfConnections = 10;
 
-    async void Awake()
```

Comment:
It is a good rule of thumb to always relay stuff right before you need it, so right before trying to create a host or joining it.

Reply:
Thank makes sense thanks for the info.